quizzes/firebase.json

```diff
@@ -0,0 +1,48 @@
+{
+    "metadata": {
```

Comment:
I think we should try to add global identifier for Quiz and Question itself like
```json
{
"metadata": {
        "uuid": "fb833dbe-c98b-45b1-b054-749a2ae77a05", //global identifier
        "title": "Firebase",
        "author": "Fahli Saputra",
        "description": "short description",
        "thumbnail": "https://fahli.net/wp-content/uploads/2020/07/Instalasi-Debian-9-dan-Konfigurasi-IP.jpg"
    },
}
```

```json
{
            "uuid": "fb833dbe-c98b-45b1-b054-749a2ae77a05" //global identifier
            "question": "What is the capital of India?",
            "image": "",
            "options": [
                {
                    "answer": "Delhi",
                    "isCorrect": true
                },
                {
                    "answer": "Mumbai"
                },
                {
                    "answer": "Kolkata"
                },
                {
                    "answer": "Chennai"
                }
            ]
        },
```

Reply:
done on commit `d11af97cb72fcbfdda8b7fa480ab181d77000186`